README.md

```diff
@@ -11,10 +11,14 @@
 
 <br />
 
+## Credit
```

Comment:
Attribution is great; I'd move it to the bottom of the README

Reply:
Good idea; the credit section has been moved accordingly.